sdk/inc/azure/core/az_http_transport.h

```diff
@@ -119,14 +128,18 @@ struct _az_http_policy
  *
  * @param[in] request HTTP request to get HTTP header from.
  * @param[in] index Index of the HTTP header to get.
- * @param[out] out_header Pointer to write the result to.
+ * @param[out] out_name Pointer to an #az_span to write the header's name.
+ * @param[out] out_value Pointer to an #az_span to write the header's value.
```

Comment:
Consistency with the complimentary response API docs.
```suggestion
 * @param[out] out_name A pointer to an #az_span to receive the header's name.
 * @param[out] out_value A pointer to an #az_span to receive the header's value.
```